packages/xchain-ethereum/src/utils.ts

```diff
@@ -92,7 +92,7 @@ export const validateAddress = (address: Address): boolean => {
  */
 export const getTokenAddress = (asset: Asset | null): string | null => {
   if (asset) {
-    const assetAddress = asset.symbol.slice(asset.ticker.length + 1)
+    const assetAddress = asset.symbol.slice(asset.ticker.length + 1).toLowerCase()
```

Comment:
In general it would be nice to create ETH/ERC20 `Asset`'s based on valid check sum addresses only. Then we would not need to use `toLowerCase()`...